core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -176,7 +178,13 @@ private int localCachedRead(PageReadTargetBuffer bytesBuffer, int length,
     if (cacheContext != null && cacheContext.getCacheIdentifier() != null) {
       pageId = new PageId(cacheContext.getCacheIdentifier(), currentPage);
     } else {
-      pageId = new PageId(Long.toString(mStatus.getFileId()), currentPage);
+      Boolean isDora =
+          Configuration.getBoolean(PropertyKey.DORA_CLIENT_READ_LOCATION_POLICY_ENABLED);
```

Comment:
This can be a field of the class

Reply:
If there are use cases somewhere else, I will move it to be class member.

Reply:
Configuration.getBoolean might cause performance regression because it's expensive.  is it possible to move this line into the constructor as @dbw9580 said?

Reply:
OK. Fixed.